plugins/systemd/systemd_status

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python3 -tt
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+# pylint: enable=invalid-name
+
+"""Munin plugin to monitor systemd service status.
+Copyright 2018, Kim B. Heino, b@bbbs.net, Foobar Oy
+License GPLv2+
+
+#%# capabilities=autoconf
+#%# family=auto
+"""
+
+import os
+import re
+import subprocess
+import sys
+
+
+STATES = (
+    'failed',
+    'dead',
+    'running',
+    'exited',
+    'active',
+    'listening',
+    'waiting',
+    'plugged',
+    'mounted',
+)
+
+
+def config():
+    """Autoconfig values."""
+    print('graph_title systemd services')
+    print('graph_vlabel Services')
+    print('graph_category processes')
+    print('graph_args --base 1000 --lower-limit 0')
+    print('graph_scale no')
+    print('graph_info Number of services in given activation state.')
+    for state in STATES:
+        print('{state}.label Services in {state} state'.format(state=state))
+    print('failed.warning 0:0')
+    if os.environ.get('MUNIN_CAP_DIRTYCONFIG') == '1':
+        fetch()
+
+
+def fetch():
+    """Print runtime values."""
+    # Get data
+    try:
+        output = subprocess.check_output([
+            '/bin/systemctl',
+            'list-units'])  # deb9/py3.5 doesn't have encoding parameter
+    except (OSError, subprocess.CalledProcessError):
+        return
+
+    # Parse data
+    states = {state: 0 for state in STATES}
+    for line in output.splitlines():
+        line = line.decode('utf-8').split()
```

Comment:
Just for my taste: I would avoid overriding an existing variable `line` with a different type of content (*line* changes to *list of line tokens*). Maybe use `tokens` instead?
Of course, this is just a matter of taste ...